packages/pyroscope-flamegraph/src/Toolbar.spec.tsx

```diff
@@ -88,7 +88,7 @@ describe('ProfileHeader', () => {
         handleSearchChange={() => {}}
         reset={() => {}}
         updateFitMode={() => {}}
-        fitMode={HeadMode}
+        fitMode={TailMode}
```

Comment:
why did you switch this from `HeadMode` to `TailMode` ?

Reply:
I thought we can test both cases here